tests/functional-all.js

```diff
@@ -16,6 +16,7 @@ define([
   "./functional/comments-non-auth.js",
   "./functional/contributors-non-auth.js",
   "./functional/comments-auth.js",
+  "./functional/image-uploads-auth.js",
```

Comment:
I decided on this name at first just to follow the pattern that I found on the naming of the files, but this is more semantic I think.